main.py

```diff
@@ -40,55 +40,63 @@ def make_menu_decision():
         else:
             print("I am sorry; I do not recognize this option")
             
-# This function will add the first and last name to empData file.
+# This function will add the first and last name to empData file. 
+# This will also create and store the email
 def create_emp():
     # TODO add logging info
     with open('empData.txt', 'a') as f:
         addFirst = input("Enter the employee's first name: ")
         addLast = input("Enter the employyee's last name: ")
         newEmp = Employee(addFirst, addLast)
         f.write(f'Name: {newEmp.full_name}, Email: {newEmp.email}\n')
-        print("Name successfully added.")
+        print("Employee successfully added.")
```

Comment:
Based on the feedback below you can have a function which provides a success result (True/False) and an error message (empty if successful, some message if not). This can determine if a success message actually shows for the end user or not...